tradesman/model_creation/zoning/zones_with_location.py

```diff
@@ -29,11 +37,11 @@ def zones_with_location(hexb, all_subdivisions):
     empties = data_complete.division_name.isna()
 
     for idx, record in data_complete[empties].iterrows():
-        geo = record.geometry
-        dscrt = [x for x in dindex.nearest(geo.bounds, 10)]
-        dist = [states.loc[d, "geometry"].distance(geo) for d in dscrt]
+        geo = box(*record.geometry.bounds)
+        dscrt = [x for x in dindex.nearest(geo, 10)]
+        dist = [states.loc[d, "geometry"].distance(geo).tolist() for d in dscrt]
```

Comment:
This is a weird syntax

`dist = [states.geometry[loc].values[0].distance(geo) for d in dscrt]`

You may not even need the `.values[0]` part